MangaManager/src/__init__.py

```diff
@@ -24,18 +23,9 @@
 SOURCES_DIR = Path(sub_mm_path, "ExternalSources")
 SOURCES_DIR.mkdir(exist_ok=True)
 
-CONFIG_PATH = Path(MM_PATH, _CONFIG_PATH)
```

Comment:
Settings as well as logging should be inside MM_PATH.
I don't recall if you handled this already.

Reply:
This code has no effect on my testing